store/sql_session_store.go

```diff
@@ -24,16 +26,19 @@ func NewSqlSessionStore(sqlStore *SqlStore) SessionStore {
 		table.ColMap("DeviceId").SetMaxSize(128)
 		table.ColMap("Roles").SetMaxSize(64)
 		table.ColMap("Props").SetMaxSize(1000)
+		table.ColMap("AccessToken").SetMaxSize(128)
```

Comment:
Should access token be sanitized on return of getSessions?  Similar to sessionId?